content/docs/refs-and-the-dom.md

```diff
@@ -31,30 +31,61 @@ Your first inclination may be to use refs to "make things happen" in your app. I
 
 ### Adding a Ref to a DOM Element
 
-React supports a special attribute that you can attach to any component. The `ref` attribute takes a callback function, and the callback will be executed immediately after the component is mounted or unmounted.
+>**Note:**
+>
+>The examples below have updated to use the new `React.createRef()` API, introduced in React 16.3.
```

Comment:
I think this will read a little weird in a few months when `createRef` is no longer new.

> The examples below use the `React.createRef()` API introduced in React 16.3.